tests/ProxyManagerTest/Signature/Exception/InvalidSignatureExceptionTest.php

```diff
@@ -14,7 +14,7 @@
  * @covers \ProxyManager\Signature\Exception\InvalidSignatureException
  * @group Coverage
  */
-class InvalidSignatureExceptionTest extends TestCase
+final class InvalidSignatureExceptionTest extends TestCase
 {
     public function testFromInvalidSignature() : void
```

Comment:
can be declared as static

Reply:
Same as above: test methods, unlikely to be made `static`